app/core/models.py

```diff
@@ -1,12 +1,19 @@
+import re
```

Comment:
What is this import?

Reply:
Regex, it is used in the password validators @KarenAJ  wrote

Reply:
Ah! looks good then.